src/lib/es2015.symbol.wellknown.d.ts

```diff
@@ -124,6 +124,15 @@ interface WeakMap<K, V>{
     readonly [Symbol.toStringTag]: "WeakMap";
 }
 
+interface Set<T> {
+    [Symbol.iterator]():IterableIterator<T>;

```

Comment:
I put it here as it uses Symbol.iterator. Though same argument can go with the return type